templates/zerver/help/import-from-slack.md

```diff
@@ -0,0 +1,50 @@
+# Import from Slack
+
+To import users and channels from Slack to Zulip please follow
+bellow steps:
+
+1. Generate Slack API Token from [here](https://api.slack.com/docs/oauth-test-tokens).
+   Test tokens are enough to import all of needed data.
+
+2. Go to **Settings** by clicking on cog from top right corner.
+
+    ![Settings](/static/images/help/cog_dropdown_settings.png)
+
+3. Scroll down to **Your bots** section.
+
+    ![Your bots](/static/images/help/settings-your-bots.png)
+
+4. Click on **Show/change your API key** button, provide your password
+   and copy generated API key.
+
+5. Fill in all settings in `api/integrations/slack/zulip_slack_config.py`:
+
+   `SLACK_TOKEN` - token from point number 1.
+
+   `ZULIP_USER` - e-mail of the user (the user that API key was generated for).
+
+   `ZULIP_KEY` - API key from point number 4.
+
+   `ZULIP_SITE` - Zulip API server URI.
+
+6. Please ensure the user has admin rights.
+
+7. If imported users are not from the same organization (in another case skip this point)
```

Comment:
We usually don't have sub steps. You should either make this into a separate section or a `tip` or `warn`. Make sure to use the `{!admin.md!}` macro here instead.